Update to python3

afaur opened the pull request:
Had issues getting this working in python2
```
IOError: [Errno 32] Broken pipe
```
After updating it to work with python3 the issues were resolved.

afaur committed:
Update to python3
Update readme

di commented:
Wow, I thought for sure that this project was long dead. Thanks for doing this!

This looks great, except it seems like the `bin` directory didn't get included in the PR. Could you add it?

As an aside, can I ask how you found this repo, and what you're using it for?

afaur commented:
Looks like `bin` folder for python is added to `.gitignore`.  Changed it to `scripts` directory, and updated readme.

Wanted something when `screenhero` does not work well, and thought it would be a good project to learn some python.

di commented:
Thanks!
